mmv1/products/oracledatabase/AutonomousDatabase.yaml

```diff
@@ -48,23 +48,39 @@ async:
     path: 'error'
     message: 'message'
 custom_code:
+  pre_delete: 'templates/terraform/pre_delete/oracledatabase_autonomous_database.go.tmpl'
 examples:
   - name: 'oracledatabase_autonomous_database_basic'
     primary_resource_id: 'myADB'
     vars:
       project: 'my-project'
       autonomous_database_id: 'my-instance'
+      deletion_protection: 'true'
```

Comment:
Have you run this test? I'd imagine it no longer passes because we are effectively preventing the deletion for the teardown of the resource

Reply:
I am overriding it later to false, in line 62.